package.json

```diff
@@ -4,16 +4,20 @@
   "description": "DB containing GP profile information",
   "main": "app.js",
   "scripts": {
-    "git-hooks": "yarn run lint && yarn run test",
+    "check-coverage": "istanbul check-coverage --config .istanbul.yml",
+    "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
+    "git-hook": "yarn run lint && yarn run generate-coverage && yarn run check-coverage",
```

Comment:
@st3v3nhunt Do you not have to add a COVERALLS_REPO_TOKEN as a Travis env var for it to update coveralls?